cni/network/multitenancy_mock.go

```diff
@@ -78,3 +78,74 @@ func (m *MockMultitenancy) GetContainerNetworkConfiguration(
 
 	return cnsResponse, *ipnet, nil
 }
+
+func (m *MockMultitenancy) GetNetworkContainersWithOrchestratorContext(
+	ctx context.Context,
+	nwCfg *cni.NetworkConfig,
+	podName string,
+	podNamespace string,
+) (*[]cns.GetNetworkContainerResponse, []net.IPNet, error) {
+	if m.fail {
+		return nil, []net.IPNet{}, errMockMulAdd
+	}
+
+	cnsResponseOne := &cns.GetNetworkContainerResponse{
+		IPConfiguration: cns.IPConfiguration{
+			IPSubnet: cns.IPSubnet{
+				IPAddress:    "192.168.0.4",
+				PrefixLength: ipPrefixLen,
+			},
+			GatewayIPAddress: "192.168.0.1",
+		},
+		LocalIPConfiguration: cns.IPConfiguration{
+			IPSubnet: cns.IPSubnet{
+				IPAddress:    "169.254.0.4",
+				PrefixLength: localIPPrefixLen,
+			},
+			GatewayIPAddress: "169.254.0.1",
+		},
+
+		PrimaryInterfaceIdentifier: "10.240.0.4/24",
+		MultiTenancyInfo: cns.MultiTenancyInfo{
+			EncapType: cns.Vlan,
+			ID:        1,
+		},
+	}
+
+	cnsResponseTwo := &cns.GetNetworkContainerResponse{
+		IPConfiguration: cns.IPConfiguration{
+			IPSubnet: cns.IPSubnet{
+				IPAddress:    "192.168.0.5",
+				PrefixLength: ipPrefixLen,
+			},
+			GatewayIPAddress: "192.168.0.1",
+		},
+		LocalIPConfiguration: cns.IPConfiguration{
+			IPSubnet: cns.IPSubnet{
+				IPAddress:    "169.254.0.4",
+				PrefixLength: localIPPrefixLen,
+			},
+			GatewayIPAddress: "169.254.0.1",
+		},
+
+		PrimaryInterfaceIdentifier: "10.240.0.4/24",
+		MultiTenancyInfo: cns.MultiTenancyInfo{
+			EncapType: cns.Vlan,
+			ID:        1,
```

Comment:
let's try to use the same examples for dual nic - i.e. let's have these 2 NCs belong to different vnets. Which means different ip space / IP / different Vlan ids. You can use the examples from the testbed VM which you created